js/options.js

```diff
@@ -58,4 +58,31 @@
             });
         });
     });
+
+    // Apply i18n
+    $(document).ready(function(){
+        // Basic Substitution
+        $('[data-i18n]').each(function(){
+            var $this = $(this);
+            $this.text(i18n($this.data('i18n')));
+        });
+
+        // Text with link to Play Store
+        var $installSignalLinkContent = $('<span>' + i18n('installSignalLink') + '</span>');
+        $installSignalLinkContent.find('a').attr({
+            class: 'link',
+            href: 'https://play.google.com/store/apps/details?id=org.thoughtcrime.securesms',
+            target: '_blank'
+        });
+        $('#installSignalLink').append($installSignalLinkContent);
+
+        // Text with link to Twitter
+        var $installFollowUsContent = $('<span>' + i18n('installFollowUs') + '</span>');
+        $installFollowUsContent.find('a').attr({
+            class: 'link',
+            href: 'https://twitter.com/whispersystems',
+            target: '_blank'
+        });
+        $('#installFollowUs').append($installFollowUsContent);
```

Comment:
You might be able to solve this elegantly by using [placeholders](https://developer.chrome.com/extensions/i18n-messages#placeholders) and/or [substitution strings](https://developer.chrome.com/extensions/i18n#method-getMessage). Our i18n wrapper should certainly be extended to expose this feature if it proves useful, but I would try keep any additional rendering logic encapsulated in the view class.